src/wallet/rpcwallet.cpp

```diff
@@ -176,7 +176,7 @@ void TxExpandedToJSON(const CWalletTransactionBase& tx,  UniValue& entry)
         const CTxOut& txout = tx.getTxBase()->GetVout()[i];
         UniValue out(UniValue::VOBJ);
         out.pushKV("value", ValueFromAmount(txout.nValue));
-        out.pushKV("valueSat", txout.nValue);
+        out.pushKV("valueZat", txout.nValue);
```

Comment:
Shall we have both JSON K/V pairs?

Reply:
We can keep `valueZat` since the Explorer will be changed accordingly.